tasks/main.yml

```diff
@@ -1,37 +1,50 @@
 ---
-# tasks file for ansible-role-git-repository
 
 - include_vars: "{{ ansible_os_family }}.yml"
 
 - include: "install-{{ ansible_os_family }}.yml"
 
-- name: Create log directory
-  file:
-    path: "{{ git-repository_log_dir }}"
-    mode: 0755
-    owner: "{{ git-repository_user }}"
-    group: "{{ git-repository_group }}"
-    state: directory
-  notify: Restart git-repository
+- name: Assert that git_clone_repositories has mandatory keys
```

Comment:
how about checking `item.state` is `present` or `absen`t here?
is there any reason for separating it as the assertion below?